consensus/src/threshold_simplex/actors/batcher/actor.rs

```diff
@@ -282,7 +282,7 @@ impl<
         let leader_index = self.supervisor.is_participant(self.view, leader)?;
         Some(
             self.notarizes[leader_index as usize].is_some()
-                || self.finalizes[leader_index as usize].is_some(),
+                || self.nullifies[leader_index as usize].is_some(),
```

Comment:
This was already handled correctly by `simplex` (and was introduced in the refactor).